releng/cody-feature/run_with_java.launch

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.pde.ui.RuntimeWorkbench">
```

Comment:
What does this launch configuration do?

Reply:
It runs eclipse with java editing features enabled but it's initialization is slightly longer, especially on the clean workspace. The difference is however small, so we may want to retire the old `minimal_run.launch`.